src/components/CalendarMonth.jsx

```diff
@@ -185,7 +194,8 @@ class CalendarMonth extends React.Component {
             verticalScrollable && styles.CalendarMonth_caption__verticalScrollable,
           )}
         >
-          <strong>{monthTitle}</strong>
+          {renderCaption && renderCaption({ month, onMonthSelect, onYearSelect })}
+          {!renderCaption && <strong>{monthTitle}</strong>}
```

Comment:
any reason not to make this be the default function for renderCaption?

Reply:
Nope, sounds great, I'll update that.